.circleci/config.yml

```diff
@@ -30,6 +30,7 @@ jobs:
               echo "set -e" >> $BASH_ENV
               echo "export OPENBLAS_NUM_THREADS=4" >> $BASH_ENV
               echo "export XDG_RUNTIME_DIR=/tmp/runtime-circleci" >> $BASH_ENV
+              echo "export MNE_3D_OPTION_MULTI_SAMPLES=1" >> $BASH_ENV
```

Comment:
This was the magic. Our detection of MESA in MNE-Python must be flawed...

Reply:
Good to know, thanks for fixing that